src/components/audio-zone.js

```diff
@@ -42,17 +49,35 @@ AFRAME.registerComponent("audio-zone", {
     this.enableDebug(this.data.debuggable && window.APP.store.state.preferences.showAudioDebugPanel);
   },
 
+  tick: (() => {
+    const quat = new THREE.Quaternion();
+    return function() {
+      this.el.object3D.quaternion.set(0, 0, 0, 1);
+      this.el.object3D.quaternion.multiply(this.el.object3D.parent.getWorldQuaternion(quat).inverse());
+      this.el.object3D.updateMatrix();
```

Comment:
To prevent unnecessarily raising the dirty flag, I would like to suggest to call `updateMatrix()` only if `quaternion` has diff with the previous values, like

```
const EPSILON = 0.00000000001;

...

this.el.object3D.parent.getWorldQuaternion(quat).inverse();
if (!this.el.object3D.quaternion.near(quat, EPSILON)) {
  this.el.object3D.quaternion.copy(quat);
  this.el.object3D.updateMatrix();
}
```

Reply:
Yes, that's more efficient. Updating.